### PR TITLE
Update docs to use self-closing tags

### LINKED ISSUE #46: Change Documentation to use Self-Closing Tags
## Summary
Now that Tram-One supports self-closing tags, we should update all the documentation and examples that don't use children to be self-closing.

### Example
In the following example we can change `<footer></footer>` to `<footer/>`
https://github.com/Tram-One/tram-one/blob/455c1fd06d7e1d3ffacd3898399be02bd62edc34/examples/using-custom-elements/chrome.js#L11-L17

### PR DESCRIPTION
**Pull request by robbawebba**
I've updated all tags in the examples and the README to be self-closing if they did not contain any inner HTML.

closes #46

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ const html = Tram.html()
 // create a custom element to display a color option
 const colorElement = (attrs, children) => {
   return html`
-    <button onclick=${attrs.onclick}>${children}</button>
+    <button onclick=${attrs.onclick}>${children} />
```
Comment (JRJurman) [inline README.md]: I think we need to keep this ending tag

File: examples/ssr-example/index.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ const home = () => html`
   <div>
     <h1>This is the Home Page</h1>
     <div> This is rendered on a server, and then served up to you! </div>
-    <div> We also have a number page here: <a href="/num">/num</a>
+    <div> We also have a number page here: <a href="/num">/num</a> </div>
```
Comment (JRJurman) [inline examples/ssr-example/index.js]: Nice catch 👍

File: README.md (hunk 1 above)
Comment (robbawebba) [reply README.md]: Whoops! Thanks for catching this one

Comment (JRJurman) [reply README.md]: NP, when you get a chance, just add a commit to this branch to re-add the closing tag.

**Comment by robbawebba** [comment]
@JRJurman The README has been updated to include your requested changes :)

**Pull request merged**